.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+### :sparkles: Description of Change
```

Comment:
This template will be auto-generated for every new PR. Feel free to suggest other things to add but maybe we can review it as a team in a tech huddle. I just wanted to have something for now